src/open_inwoner/accounts/tests/test_backends.py

```diff
@@ -69,3 +75,108 @@ def test_admin_oidc_use_correct_backend(
         result = auth.authenticate(request)
 
         self.assertEqual(result, self.user)
+
+
+@override_settings(
+    AUTHENTICATION_BACKENDS=[
+        "open_inwoner.accounts.backends.UserModelEmailBackend",
+    ]
+)
+class UserModelEmailBackendTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.password = "keepitsecert"
+        cls.user = UserFactory(
+            login_type=LoginTypeChoices.default, password=cls.password
+        )
+        for login_type in LoginTypeChoices:
+            UserFactory(login_type=login_type)
+
+    def test_duplicate_emails_on_case_results_in_no_match(self):
+        request = RequestFactory().post(reverse("login"))
+
+        UserFactory(email=self.user.email.upper(), login_type=self.user.login_type)
+        result = auth.authenticate(
+            request, username=self.user.email, password=self.password
+        )
+        self.assertEqual(result, None)
+
+    def test_correct_username_password_return_user(self):
+        request = RequestFactory().post(reverse("login"))
+
+        result = auth.authenticate(
+            request, username=self.user.email, password=self.password
+        )
+        self.assertEqual(result, self.user)
+
+    def test_incorrect_username_password_return_none(self):
+        request = RequestFactory().post(reverse("login"))
+
+        for username, password in (
+            (self.user.email, "incorrect"),
+            ("incorrect", self.password),
+        ):
+            result = auth.authenticate(request, username=username, password=password)
+            self.assertEqual(result, None)
+
+    def test_missing_username_and_or_password_returns_none(self):
+        for username in (self.user.email, "", None):
+            for password in (self.password, "", None):
+                if username and password:
+                    # This is the successful case, exclude it, but ensure we
+                    # also have permutations with one valid/one invalid value.
+                    continue
+
+                with self.subTest(f"{username=} {password=}"):
+                    request = RequestFactory().post(reverse("login"))
+                    result = auth.authenticate(
+                        request, username=username, password=password
+                    )
+                    self.assertEqual(result, None)
+
+
+@override_settings(
+    AUTHENTICATION_BACKENDS=[
+        "open_inwoner.accounts.backends.Verify2FATokenBackend",
+    ]
+)
+class Verify2FATokenBackendTestCase(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.user = UserFactory(login_type=LoginTypeChoices.default)
+        cls.expires_in = getattr(settings, "ACCOUNTS_USER_TOKEN_EXPIRE_TIME", 300)
+        cls.make_token = lambda: totp(cls.user.seed, period=cls.expires_in)
```

Comment:
```suggestion
        cls.token = lambda: totp(cls.user.seed, period=cls.expires_in)
```
The value for this will be the same for all tests, so it's a bit misleading to call it `make_token` (which to me suggests that a new token is made every time this is called)